Formula/watchexec.rb

```diff
@@ -0,0 +1,18 @@
+class Watchexec < Formula
+  desc "Execute commands when watched files change"
+  homepage "https://github.com/mattgreen/watchexec"
+  url "https://github.com/mattgreen/watchexec/archive/1.6.0.tar.gz"
+  sha256 "0ce7b26bbc05400a835b5a06e13ef69aac90a6d93ef4b989241258494f3c967f"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/watchexec"
+    man1.install "doc/watchexec.1"
+  end
+
+  test do
+    system "#{bin}/watchexec", "--version"
```

Comment:
I can add an option (such as `-1`) to support stopping after a single change is detected. This would clean up the test well and mitigate potential issues with the `sleep` call not being long enough if you're unlucky.

It'd be good for integration testing for myself, as well!

Tracking issue: https://github.com/mattgreen/watchexec/issues/35